apps/frontend/components/global/Navigation/DesktopNavigation.tsx

```diff
@@ -112,7 +113,7 @@ export function DesktopNavigationRight(props: {
 }) {
   return (
     <div className="hidden gap-3 lg:flex lg:items-center lg:justify-center">
-      {props.items?.map((item, index) => (
+      {/* {props.items?.map((item, index) => (
```

Comment:
Is this commented intentionally?

Reply:
yes, that was the "Registry" button on the top-right in the nav bar

Reply:
but Mohab wants it back actually. i will uncomment it